app/src/controllers/launch/sagas.js

```diff
@@ -24,12 +24,15 @@ function* fetchLaunchesWithParams({ payload }) {
   );
 }
 
+const notEmptyConditionsPredicate = ({ value }) =>
```

Comment:
It is similar with [this](https://github.com/reportportal/service-ui/pull/1230/files#diff-31c78ba55f47f72427159c7b26a8fdf3R18)
May be better to create `.utils.js` and hold it there also with [this](https://github.com/reportportal/service-ui/pull/1230/files#diff-31c78ba55f47f72427159c7b26a8fdf3R20)

Reply:
fixed